### PR TITLE
MAUI 829675 - UG documentation for Zoom in MAUI Image Editor

### PR DESCRIPTION
**Pull request by Shalini-Ashokan**
Hi @karthikraja-arumugam,

 I have prepared UG content for zoom  in Maui image editor control. Could you please review it and let me know if you have any concerns?

Regrads,
Shalini.

### CODE REVIEW

File: MAUI/ImageEditor/save.md [hunk 1]
```diff
@@ -213,4 +213,108 @@ The [`ImageSaved`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEdit
 
 {% endtabs %}
 
+## Reset
+
+The [`Reset`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_Reset) method, resets an image to its original state by discarding the changes performed to it. You can start fresh and explore different creative possibilities.the complete set of changes made in image and also resets the image to original loaded image.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/save.md:218]: You can start fresh and explore different creative possibilities. The complete set of changes made in the image and also resets the image to the original loaded image.

File: MAUI/ImageEditor/save.md [hunk 2]
```diff
@@ -213,4 +213,108 @@ The [`ImageSaved`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEdit
...
+    {
+        this.imageEditor.Reset();
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Reset events
+
+The SfImageEditor has events when performing reset operation namely [`BeginReset`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_BeginReset).
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/save.md:244]: The SfImageEditor has event when performing reset operation namely [`BeginReset`]

File: MAUI/ImageEditor/save.md (hunk 1 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/save.md]: Updated a content

File: MAUI/ImageEditor/save.md (hunk 2 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/save.md]: Changed a content

File: MAUI/ImageEditor/migration.md [hunk 3]
```diff
@@ -516,4 +516,13 @@ Applicable for crop view selection, shape and text annotation selection.
 <td>{{'[ImageEffect](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_ImageEffect_Syncfusion_Maui_ImageEditor_ImageEffect_System_Nullable_System_Double__)'| markdownify }}</td>
 <td>Method to apply various visual effects to the image, such as adjusted brightness, blur, contrast, exposure, saturation, sharpen, hue, and opacity.</td>
 </tr>
-</table>
\ No newline at end of file
+</table>
+
+## Upcoming features
+
+* Support to browse image from local device storage.
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/migration.md:523]: * Built in support to browse image from local device storage.
* Built in support to save image using folder picker in device storage.
* Localization support for the image editor toolbar, allowing you to use it in your preferred language.
* Support for annotations Z ordering.
* Support for serialize and deserialize the annotations. 
* Support to add custom shapes or views. 
* Support for pan and zoom improvements while performing the crop action on images.
* Support to toolbar customization.

File: MAUI/ImageEditor/zooming.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title: Zooming in .NET MAUI ImageEditor Control | Syncfusion
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/zooming.md:3]: title: Zoom image using .NET MAUI Image Editor Control | Syncfusion
description: Learn here all about how to zoom image in Syncfusion .NET MAUI Image Editor(SfImageEditor) control and more.

File: MAUI/ImageEditor/zooming.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+description: Learn here all about how to Zooming in Syncfusion .NET MAUI ImageEditor(SfImageEditor) control and more.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Zooming in .NET MAUI ImageEditor (SfImageEditor)
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/zooming.md:10]: .NET MAUI Image Editor (SfImageEditor) Image Zooming

File: MAUI/ImageEditor/zooming.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+The image editor control in .NET MAUI provides support for zooming and panning actions on an image.
+
+The following properties are used for zooming feature of the image editor control:
+
+* EnableZooming
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/zooming.md:16]: * EnableZooming add proper AllowZoom API

File: MAUI/ImageEditor/zooming.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Zooming in .NET MAUI ImageEditor (SfImageEditor)
+
+The image editor control in .NET MAUI provides support for zooming and panning actions on an image.
+
+The following properties are used for zooming feature of the image editor control:
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/zooming.md:14]: Add link for API The following properties

File: MAUI/ImageEditor/zooming.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+
+* EnableZooming
+* ZoomLevl
+* Maximum ZoomLevel
+
+## Enable zooming
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/zooming.md:20]: ## Allow image zooming

File: MAUI/ImageEditor/zooming.md [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+     SfImageEditor imageEditor = new SfImageEditor();
+     imageEditor.Source = "image.jpeg";
+     imageEditor.AllowZoom = false;
+     this.content = imageEditor;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Zoom level
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/zooming.md:46]: ### Change image zoom level

File: MAUI/ImageEditor/zooming.md [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+     imageEditor.ZoomLevel = 2;
+     this.content = imageEditor;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Zooming in .NET Maui ImageEditor](images/zoom/imageeditor-zooming.gif)
+
+## Maximum zoom level
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/zooming.md:74]: change 3#
### Change image maximum zoom level

File: MAUI/ImageEditor/zooming.md (hunk 7 above)
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/zooming.md:14]: remove this
The following properties are used for zooming feature of the image editor control:

* EnableZooming
* ZoomLevl
* Maximum ZoomLevel

File: MAUI/ImageEditor/save.md [hunk 11]
```diff
@@ -213,4 +213,108 @@ The [`ImageSaved`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEdit
...
+    private void OnBeginReset(object sender, CancelEventArgs e)
+    {
+         e.Cancel = true;
+    }
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## ImageLoaded Event
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/save.md:269]: ## Image loaded event

File: MAUI/ImageEditor/save.md [hunk 12]
```diff
@@ -213,4 +213,108 @@ The [`ImageSaved`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEdit
...
+
+    private void OnResetClicked(object sender, EventArgs e)
+    {
+        this.imageEditor.Reset();
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Reset events
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/save.md:242]: ## Reset image event

File: MAUI/ImageEditor/save.md [hunk 13]
```diff
@@ -213,4 +213,108 @@ The [`ImageSaved`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEdit
...
+
+The SfImageEditor has event when performing reset operation namely [`BeginReset`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_BeginReset).
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/save.md:244]: Remove this

The SfImageEditor has event when performing reset operation namely [`BeginReset`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_BeginReset).

### BeginReset

File: maui-toc.html [hunk 14]
```diff
@@ -462,11 +462,12 @@
             <li><a href="/maui/ImageEditor/overview">Overview</a></li>
             <li><a href="/maui/ImageEditor/getting-started">Getting Started</a></li>
             <li><a href="/maui/ImageEditor/migration">Migrate from Xamarin.Forms</a></li>
-            <li><a href="/maui/ImageEditor/save">Save and Events</a></li>
-            <li><a href="/maui/ImageEditor/shape">Shapes</a></li>
             <li><a href="/maui/ImageEditor/text">Text</a></li>
+            <li><a href="/maui/ImageEditor/shape">Shapes</a></li>
```
Comment (Jeyasri-Murugan) [inline maui-toc.html:466]: Change order as like FT

File: MAUI/ImageEditor/save.md [hunk 15]
```diff
@@ -213,4 +213,108 @@ The [`ImageSaved`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEdit
 
 {% endtabs %}
 
+## Reset
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/save.md:216]: split Reset in separate md file

Reset image using .NET MAUI Image Editor (SfImageEditor)

File: MAUI/ImageEditor/zooming.md [hunk 16]
```diff
@@ -0,0 +1,98 @@
...
+
+{% tabs %}
+
+{% highlight XAML %}
+  <ContentPage 
+            . . .
+            <imageEditor:SfImageEditor Source="image.jpeg" ZoomLevel="2"/>
+
+   </ContentPage>
+
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/zooming.md:58]: Add different way to load image source in Getting started title as discussed

Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/zooming.md:52]: Added a different way to load image source

File: MAUI/ImageEditor/save.md (hunk 15 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/save.md]: Added a separate file for reset

File: maui-toc.html (hunk 14 above)
Comment (Shalini-Ashokan) [reply maui-toc.html]: Changed a order as like FT

File: MAUI/ImageEditor/migration.md (hunk 3 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/migration.md]: Updated a upcoming features

File: MAUI/ImageEditor/zooming.md (hunk 4 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/zooming.md]: Updated a title and description

File: MAUI/ImageEditor/zooming.md (hunk 5 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/zooming.md]: Updated a title

File: MAUI/ImageEditor/zooming.md (hunk 8 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/zooming.md]: Changed

File: MAUI/ImageEditor/zooming.md (hunk 9 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/zooming.md]: Changed

File: MAUI/ImageEditor/zooming.md (hunk 10 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/zooming.md]: Changed 3#

File: MAUI/ImageEditor/zooming.md (hunk 7 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/zooming.md]: Removed

File: MAUI/ImageEditor/save.md (hunk 11 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/save.md]: Changed a title

File: MAUI/ImageEditor/save.md (hunk 12 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/save.md]: Changed a title

File: MAUI/ImageEditor/save.md (hunk 13 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/save.md]: Removed

File: MAUI/ImageEditor/events.md [hunk 17]
```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: Events in .NET MAUI ImageEditor control | Syncfusion
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/events.md:3]: use Image Editor instead ImageEditor in all places

File: MAUI/ImageEditor/getting-started.md [hunk 18]
```diff
@@ -97,14 +97,18 @@ Refer to the following steps to add an image to the project:
 
 N> Image formats such as JPEG and PNG can be loaded to the image editor.
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/getting-started.md:98]: is this step common to add types, if its particular move under
image

File: MAUI/ImageEditor/getting-started.md [hunk 19]
```diff
@@ -120,7 +124,101 @@ public partial class MainPage : ContentPage
...
+        InitializeComponent();
+        SfImageEditor imageEditor = new SfImageEditor();
+        imageEditor.Source = ImageSource.FromUri(new Uri("https://dummyimage.com/300x200/000/fff.png"));
+        this.content = imageEditor;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+### Load a resource file
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/getting-started.md:166]: Load an image from Resource folder

File: MAUI/ImageEditor/getting-started.md [hunk 20]
```diff
@@ -120,7 +124,101 @@ public partial class MainPage : ContentPage
...
         SfImageEditor imageEditor = new SfImageEditor();
-        imageEditor.Source = "image.jpeg";
+        imageEditor.Source = ImageSource.FromFile("D:\\images\\image.jpeg");
+        this.content = imageEditor;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+### Load a remote file
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/getting-started.md:134]: ### Load an image from URI

File: MAUI/ImageEditor/getting-started.md [hunk 21]
```diff
@@ -120,7 +124,101 @@ public partial class MainPage : ContentPage
...
+        InitializeComponent();
+        SfImageEditor imageEditor = new SfImageEditor();
+        imageEditor.Source =  ImageSource.FromResource("MyProject.Resources.Images.image.jpeg");
+        this.content = imageEditor;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+### Loading from stream
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/getting-started.md:198]: ### Load an image from stream

File: MAUI/ImageEditor/zooming.md [hunk 22]
```diff
@@ -0,0 +1,92 @@
...
+     imageEditor.ZoomLevel = 2;
+     this.content = imageEditor;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Zooming in .NET Maui ImageEditor](images/zoom/imageeditor-zooming.gif)
+
+### Maximum zoom level
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/zooming.md:68]: ### Change maximum zoom level

File: MAUI/ImageEditor/zooming.md [hunk 23]
```diff
@@ -0,0 +1,92 @@
...
+     SfImageEditor imageEditor = new SfImageEditor();
+     imageEditor.Source = "image.jpeg";
+     imageEditor.AllowZoom = false;
+     this.content = imageEditor;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Change image zoom level
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/zooming.md:40]: change 2#

Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/zooming.md]: Changed 2#

File: MAUI/ImageEditor/zooming.md (hunk 22 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/zooming.md]: Changed a heading

File: MAUI/ImageEditor/getting-started.md (hunk 21 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/getting-started.md]: Changed a heading

File: MAUI/ImageEditor/getting-started.md (hunk 20 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/getting-started.md]: Changed a heading

File: MAUI/ImageEditor/getting-started.md (hunk 19 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/getting-started.md]: Changed a heading

File: MAUI/ImageEditor/events.md (hunk 17 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/events.md]: Changed in all places

File: MAUI/ImageEditor/getting-started.md (hunk 18 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/getting-started.md:90]: Moved a resource topic below

File: MAUI/ImageEditor/events.md [hunk 24]
```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: Events in .NET MAUI Image Editor control | Syncfusion
```
Comment (M-A32) [inline MAUI/ImageEditor/events.md:3]: Title: Events in .NET MAUI Image Editor Control | Syncfusion

File: MAUI/ImageEditor/events.md [hunk 25]
```diff
@@ -0,0 +1,65 @@
...
+description: Learn here all about the Events support in Syncfusion .NET MAUI ImageEditor(SfImageEditor) control and more.
```
Comment (M-A32) [inline MAUI/ImageEditor/events.md:4]: description: Learn here all about the Events support in the Syncfusion .NET MAUI ImageEditor(SfImageEditor) control and more.

File: MAUI/ImageEditor/events.md [hunk 26]
```diff
@@ -0,0 +1,65 @@
...
+documentation: ug
+---
+
+# Events in .NET MAUI Image Editor (SfImageEditor)
+
+The SfImageEditor supports the [`Imageloaded`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_ImageLoaded) and [`AnnotationSelected`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_AnnotationSelected) events to interact with [.NET MAUI Image Editor](https://help.syncfusion.com/maui/imageeditor/overview).
+
+## Image loaded event
+
+This [`Imageloaded`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_ImageLoaded) event will be triggered after the image has been loaded. By this event you can add any shapes or text over an image or crop an image while initially loading the image.
```
Comment (M-A32) [inline MAUI/ImageEditor/events.md:16]: This [`Imageloaded`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_ImageLoaded) event will be triggered after the image has been loaded. By this event, you can add any shapes or text over an image or crop an image while initially loading the image.

File: MAUI/ImageEditor/getting-started.md [hunk 27]
```diff
@@ -85,6 +85,78 @@ namespace GettingStarted
...
+        imageEditor.Source = ImageSource.FromFile("D:\\images\\image.jpeg");
+        this.content = imageEditor;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+### Load an image from URI
+
+To load an image from a remote URI, you can use the following code example:
```
Comment (M-A32) [inline MAUI/ImageEditor/getting-started.md:126]: To load an image from a remote URI, you can use the following code example.

File: MAUI/ImageEditor/getting-started.md [hunk 28]
```diff
@@ -93,14 +165,12 @@ Refer to the following steps to add an image to the project:
 4. Browse to the location on your computer where the image file is stored.
 5. Select the image file you want to add to the "Resources" folder.
 6. Click "Add" to add the image file to the project.
-7. Assign the image name, including its extension, to the [Source](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_Source) property of the image editor control. 
-
-N> Image formats such as JPEG and PNG can be loaded to the image editor.
+7. Assign the image name, including its extension, to the [Source](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_Source) property of the image editor control.
 
-The following code shows adding an image to the image editor control with the format as "JPEG" and name as "image."
+The following code shows adding an image to the image editor control:
```
Comment (M-A32) [inline MAUI/ImageEditor/getting-started.md:170]: The following code shows adding an image to the image editor control.

File: MAUI/ImageEditor/migration.md [hunk 29]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Migrate from Xamarin to .NET MAUI ImageEditor (SfImageEditor) | Syncfusion
+title: Migrate from Xamarin to .NET MAUI Image Editor (SfImageEditor) | Syncfusion
```
Comment (M-A32) [inline MAUI/ImageEditor/migration.md:3]: Title: Migrate from Xamarin to .NET MAUI Image Editor (SfImageEditor) | Syncfusion

File: MAUI/ImageEditor/reset.md [hunk 30]
```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: Reset image .NET MAUI Image Editor control | Syncfusion
+description: Learn about reset support in the Syncfusion .NET MAUI ImageEditor(SfImageEditor) control and more.
```
Comment (M-A32) [inline MAUI/ImageEditor/reset.md:4]: Description: Learn about reset support in the Syncfusion .NET MAUI ImageEditor(SfImageEditor) control and more.

File: MAUI/ImageEditor/save.md [hunk 31]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Save image .NET MAUI ImageEditor control | Syncfusion
+title: Save image .NET MAUI Image Editor control | Syncfusion
```
Comment (M-A32) [inline MAUI/ImageEditor/save.md:3]: Title: Save image .NET MAUI Image Editor Control | Syncfusion

File: MAUI/ImageEditor/shape.md [hunk 32]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Shape Annotations in .NET MAUI ImageEditor control | Syncfusion
+title: Shape Annotations in .NET MAUI Image Editor control | Syncfusion
```
Comment (M-A32) [inline MAUI/ImageEditor/shape.md:3]: Title: Shape Annotations in .NET MAUI Image Editor Control | Syncfusion

File: MAUI/ImageEditor/transformation.md [hunk 33]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Transform in .NET MAUI ImageEditor Control | Syncfusion
+title: Transform in .NET MAUI Image Editor Control | Syncfusion
```
Comment (M-A32) [inline MAUI/ImageEditor/transformation.md:3]: Title: Transform in .NET MAUI Image Editor Control | Syncfusion

**Review by Jeyasri-Murugan — APPROVED** [review]

File: MAUI/ImageEditor/zooming.md [hunk 34]
```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: Zoom image using .NET MAUI Image Editor Control | Syncfusion
```
Comment (M-A32) [inline MAUI/ImageEditor/zooming.md:3]: Title: Zoom Image Using .NET MAUI Image Editor Control | Syncfusion

File: MAUI/ImageEditor/zooming.md [hunk 35]
```diff
@@ -0,0 +1,92 @@
...
+description: Learn here all about how to zoom image in Syncfusion .NET MAUI Image Editor(SfImageEditor) control and more.
```
Comment (M-A32) [inline MAUI/ImageEditor/zooming.md:4]: Description: Learn here all about how to zoom image in the Syncfusion .NET MAUI Image Editor(SfImageEditor) control and more.

File: MAUI/ImageEditor/zooming.md [hunk 36]
```diff
@@ -0,0 +1,92 @@
...
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# .NET MAUI Image Editor (SfImageEditor) Image Zooming
+
+The image editor control in .NET MAUI provides support for zooming and panning actions on an image.
```
Comment (M-A32) [inline MAUI/ImageEditor/zooming.md:12]: The image editor control in the .NET MAUI provides support for zooming and panning actions on an image.

File: MAUI/ImageEditor/zooming.md [hunk 37]
```diff
@@ -0,0 +1,92 @@
...
+     imageEditor.AllowZoom = false;
+     this.content = imageEditor;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Change image zoom level
+
+Programmatically, you can zoom the loaded image without any interaction by setting a value to [`ZoomLevel`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_ZoomLevel) property.
```
Comment (M-A32) [inline MAUI/ImageEditor/zooming.md:42]: Programmatically, you can zoom the loaded image without any interaction by setting a value to the [`ZoomLevel`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_ZoomLevel) property.

**Review by M-A32 — CHANGES_REQUESTED** [review]

File: MAUI/ImageEditor/zooming.md [hunk 38]
```diff
@@ -0,0 +1,92 @@
...
+     imageEditor.ZoomLevel = 2;
+     this.content = imageEditor;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Zooming in .NET Maui ImageEditor](images/zoom/imageeditor-zooming.gif)
+
+### Change maximum zoom level
```
Comment (M-A32) [inline MAUI/ImageEditor/zooming.md:68]: ### Change the maximum zoom level

File: MAUI/ImageEditor/zooming.md [hunk 39]
```diff
@@ -0,0 +1,92 @@
...
+     SfImageEditor imageEditor = new SfImageEditor();
+     imageEditor.Source = "image.jpeg";
+     imageEditor.AllowZoom = false;
+     this.content = imageEditor;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Change image zoom level
```
Comment (M-A32) [inline MAUI/ImageEditor/zooming.md:40]: ## Change the image zoom level

File: MAUI/ImageEditor/reset.md [hunk 40]
```diff
@@ -0,0 +1,59 @@
...
+    {
+        this.imageEditor.Reset();
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+![Reset Image in .NET Maui ImageEditor](images/reset/imageeditor-reset-image.gif)
+
+## Reset image event
```
Comment (M-A32) [inline MAUI/ImageEditor/reset.md:38]: ## Reset the image event

File: MAUI/ImageEditor/reset.md [hunk 41]
```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: Reset image .NET MAUI Image Editor control | Syncfusion
+description: Learn about reset support in the Syncfusion .NET MAUI ImageEditor(SfImageEditor) control and more.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Reset image using .NET MAUI Image Editor (SfImageEditor)
```
Comment (M-A32) [inline MAUI/ImageEditor/reset.md:10]: # Reset image using the .NET MAUI Image Editor (SfImageEditor)

File: MAUI/ImageEditor/events.md [hunk 42]
```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: Events in .NET MAUI Image Editor control | Syncfusion
+description: Learn here all about the Events support in Syncfusion .NET MAUI ImageEditor(SfImageEditor) control and more.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Events in .NET MAUI Image Editor (SfImageEditor)
```
Comment (M-A32) [inline MAUI/ImageEditor/events.md:10]: # Events in the .NET MAUI Image Editor (SfImageEditor)

File: MAUI/ImageEditor/zooming.md [hunk 43]
```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: Zoom image using .NET MAUI Image Editor Control | Syncfusion
+description: Learn here all about how to zoom image in Syncfusion .NET MAUI Image Editor(SfImageEditor) control and more.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# .NET MAUI Image Editor (SfImageEditor) Image Zooming
```
Comment (M-A32) [inline MAUI/ImageEditor/zooming.md:10]: #The  .NET MAUI Image Editor (SfImageEditor) Image Zooming

**Pull request merged**